dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewManagerScope.kt

```diff
@@ -72,6 +75,22 @@ internal class RumViewManagerScope(
 
     // region Internal
 
+    @WorkerThread
+    internal fun startApplicationLaunchView(event: RumRawEvent, writer: DataWriter<Any>) {
+        val applicationLaunchViewTimeNs = appStartTimeProvider.appStartTimeNs
+        val applicationLaunchViewTime = Time(
+            timestamp = TimeUnit.NANOSECONDS.toMillis(applicationLaunchViewTimeNs),
+            nanoTime = applicationLaunchViewTimeNs
+        )
+        val viewScope = createAppLaunchViewScope(applicationLaunchViewTime)
+        viewScope.handleEvent(
+            RumRawEvent.ApplicationStarted(event.eventTime, applicationLaunchViewTimeNs),
+            writer
+        )
+        childrenScopes.add(viewScope)
+        applicationDisplayed = true
```

Comment:
should we pull this line up? just in case if inner `handleEvent` also access this flag at some point.